tests/src/Search/Web/ExampleWebTest.php

```diff
@@ -33,7 +33,7 @@ public function testElasticSearchIndex()
                 ],
```

Comment:
Don't know the effort required but it would save lines of code to maintain on the test side. I am not sure if the duplicate serialization (these yaml configuration) is necessary for correct indexing of the data.